src/Library/demos/Label/main.js

```diff
@@ -0,0 +1,63 @@
+import Gtk from "gi://Gtk";
+import Adw from "gi://Adw";
+
+const label = workbench.builder.get_object("label");
+const justification_row = workbench.builder.get_object("justification_row");
+const style_row = workbench.builder.get_object("style_row");
+const xalign_spin_button = workbench.builder.get_object("xalign_spin_button");
+const single_line_switch = workbench.builder.get_object("single_line_switch");
+
+const style_classes = [
+  "none",
+  "title-1",
+  "title-2",
+  "title-3",
+  "title-4",
+  "monospace",
+  "accent",
+  "success",
+  "warning",
+  "error",
+  "heading",
+  "body",
+  "caption-heading",
+  "caption",
+];
+
+const short_label =
+  "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore disputandum putant.";
+
+const long_label =
+  "     Lorem ipsum dolor sit amet, consectetur adipiscing elit,\n\
+  sed do eiusmod tempor incididunt ut labore et dolore magnam aliquam quaerat voluptatem.\n\
+  Ut enim mortis metu omnis quietae vitae status perturbatur,\n\
+  et ut succumbere doloribus eosque humili animo inbecilloque ferre miserum est,\n\
+  ob eamque debilitatem animi multi parentes, multi amicos, non nulli patriam,\n\
+  plerique autem se ipsos penitus perdiderunt, sic robustus animus et excelsus omni.";
+
+label.label = short_label;
```

Comment:
Another way to do this might be:

```
const short_label = long_label.replace(/\n/g, "");
```

But then the demo gets *really* wide, so maybe not. Probably okay like this :)